inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,86 +1,412 @@
+
+image:getting_started_tagset_create.png[image,width=1366,height=768]
+
+[[link_to_a_layer_and_feature]]
+** In order to use the tagset, it is necessary to *link it to a layer and feature*. Herefore, click on the *Layers*-tab and select the layer from the list at the left. As an example, let's select the layer _Named entity_. Two new panels open now: "Layer Details" and "Features". We focus on the second one. Choose the feature your tagset is made for. In this example, we choose the feature _value_. When you click on it, the panel "Feature details" opens. In this panel, scroll down to "Tagsets" and chose your tagset (to stick with our example: "Example_Tagset") from the dropdown and click _save_. The tagset which was selected before is not linked to the layer any more but the new one is. 
+
+** From now on, you can select your tags for annotating. Navigate to the annotation page (click _INCEpTION_ on the top left -> _Annotation_ and choose the document "pets2.txt"). When you double-click on any part in the text, for example "Socke" in line one, and click on the dropdown _value_ on the right, you find the tags "DOG" and "CAT" to choose from. (For details on how to annotate, see <<First Annotations with {product-name}>>)
+
+** You might want to xref:link_to_a_layer_and_feature[link "Named Entity tags" again to the _Named entity_ Layer and _value_ feature] in order to use them like they were before our little experiment.
+
+image:getting_started_tagset_use.png[image,width=1285,height=574]
+
+
+** For more details on Tagsets, see the main documentation, <<Tagsets>>.
+
+*_Note_: Tagsets can be changed and deleted. But the annotations they have been used for will remain with the same tag though.* Other than the defalut layers, default tagsets can also be deleted.
+
+
+NOTE: *Saving:* Some steps, like annotations, are saved automatically in {product-name}. Others need to be saved manually. Whenever there is a blue *save*-button, it is necessary to click it to save the work.
+
+[[knowledge_bases_in_getting_started]]
+NOTE: *Knowledge Bases* are data bases for knowledge. Let's assume, the mention "Paris" is to be annotated. There are many different Parises - persons, the capital city of France and more - so the annotation is to tell clearly what entity named Paris is meant here. Herefore, the knowledge base needs to have an entry of the correct entity. In the annotation, we then want to make a reference to that very entry. +
```

Comment:
- "herefore": I checked again, it's fine. what is the problem why you would not use it?
- "You can also create your own...": I don't understand. What do you mean?

Reply:
- "herefore": Ok, if it is widely used. I had not heard the word before and thought it might be a literal translation of the German "hierfür"
- One sentence down, you use "One can also create own" -> I would use "you" because we did that before and "your own" instead of just "own"